public/app/plugins/panel/geomap/GeomapPanel.tsx

```diff
@@ -250,7 +250,7 @@ export class GeomapPanel extends Component<Props, State> {
     pointerMoveListener(evt, this);
   };
 
-  initMapView = (config: MapViewConfig, sharedView?: View | undefined): Array<View | undefined> => {
+  initMapView = (config: MapViewConfig): Array<View | undefined> => {
     let view = new View({
       center: [0, 0],
       zoom: 1,
```

Comment:
On https://github.com/grafana/grafana/blob/1a58a99f37ffcef59b8618c57efb7440e0dd13a9/public/app/plugins/panel/geomap/GeomapPanel.tsx#L270 is it still needed to return both sharedView + view here?